### ISSUE TITLE
Registry viewer shows only default version for multi-version stacks

### ISSUE THREAD

**Issue by kadel** [devfile/api#995]
## Which area this feature is related to?

/kind bug

### Which area this bug is related to?

/area registry

## Bug Summary

`go`, `java-springboot`, and `python` stacks have multiple versions but the other versions are not displayed anywhere in the Devfile viewer (https://registry.devfile.io/viewer/devfiles/Community+java-springboot)

### Suggestion on how to fix the bug
The Viewer should show all available Stack versions and its content with information which version is default  one.

**Comment by johnmcollier** [comment]
HI @kadel, the current registry viewer doesn't support multi-version stacks, which is why only the default stack version is returned.

This will be fixed by https://github.com/devfile/api/issues/950 and https://github.com/devfile/api/issues/747 once https://github.com/devfile/registry/pull/149 is merged (but we're currently hitting some issues on 4.12)

**Comment by johnmcollier** [comment]
Closing in favour of https://github.com/devfile/api/issues/950
